server/services/conditionService.test.ts

```diff
@@ -0,0 +1,730 @@
+import { AdditionalCondition, AdditionalConditionAp } from '../@types/licenceApiClientTypes'
```

Comment:
This is the existing test - an integration test, testing basically the same code at a higher level proving it still works.
This can be rewritten to mock out (so decouple) the formatter

We're lacking test coverage for other public methods on this service